Modified content to adhere better to IBM style guide (#65)
devfile/docs: docs/modules/user-guide/partials/proc_adding-a-name-to-a-devfile.adoc

= Adding a name to a devfile

[role="_abstract"]

> Adding a name to a devfile is mandatory. Both `name` and `generateName` are optional attributes, but you must define at least one of them..

Reviewer: ```suggestion
Adding a name to a devfile is mandatory. Use the `name` or `generateName` attributes (or both) to define the name.
```
